doc/Settings.md

```diff
@@ -141,7 +141,7 @@ The `defaultInstallRoot` affects the install location when a package requires on
 
 ```json
     "installBehavior": {
-        "defaultInstallRoot": "C:\installRoot"
+        "defaultInstallRoot": "C:/installRoot"
```

Comment:
`\` probably is not correct here, but Windows uses backslashes for paths, so I think replacing with `\\` would be more accurate.